axolotl-web/src/store/store.js

```diff
@@ -1,11 +1,9 @@
-import Vuex from 'vuex'
-import Vue from 'vue'
+import { createStore } from 'vuex'
 import { router } from '../router/router';
-import {validateUUID} from '@/helpers/uuidCheck'
+import { validateUUID } from '@/helpers/uuidCheck'
+import main from "../main";
```

Comment:
Of course the import can be called whatever we want due to the default export, but what about keeping the same name as for the createApp call, and also here use `app`? One name for the same thing, might ease the mental mapping required.